src/components/DragDrop/DragDrop.module.css

```
@@ -111,23 +110,23 @@
   }
 
   /**
-  * 1) When there are more than two containers, apply a border
-  */
+    * 1) When there are more than two containers, apply a border
+    */
```

Oops 2 spaces indentation here.
```suggestion
   * 1) When there are more than two containers, apply a border
   */
```